### PR TITLE
Add CCLW stats to the config endpoint

### PR DESCRIPTION
**Pull request by diversemix**
# Description
Adds a breakdown of CCLW stats for use on the homepage 
See PDCT-553

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] The PR represents a single feature (small driveby fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: app/api/api_v1/schemas/metadata.py [hunk 1]
```diff
@@ -6,6 +6,14 @@
 TaxonomyData = Mapping[str, Mapping[str, Union[bool, Sequence[str]]]]
 
 
+class OrganisationStats(BaseModel):
+    """Definition of stats used on homepage"""
+
+    total: int
+    laws: int
+    policies: int
```
Comment (PatFawbertMills) [inline app/api/api_v1/schemas/metadata.py:14]: Could we also get the number of `UNFCCC` type?

Comment (diversemix) [reply app/api/api_v1/schemas/metadata.py:15]: Done- tho I did rejig the structure to now include "organisations" so everything pertaining to an org is under there :smile:

File: app/core/organisation.py [hunk 2]
```diff
@@ -60,3 +66,30 @@ def get_organisation_taxonomy_by_name(db: Session, org_name: str) -> TaxonomyDat
...
+            db.query(Family)
+            .join(
+                FamilyOrganisation,
+                Family.import_id == FamilyOrganisation.family_import_id,
+            )
+            .filter(FamilyOrganisation.organisation_id == org.id)
+            .filter(Family.family_category == category)
+            .count()
+        )
+        count_by_category[category] = count
```
Comment (olaughter) [inline app/core/organisation.py:89]: Wonder if this will scale, we might want to look at using a group by instead?

**Review by olaughter — APPROVED** [review]
lgtm, one thing I'm unsure about below

File: app/core/organisation.py (hunk 2 above)
Comment (diversemix) [reply app/core/organisation.py]: done

File: app/core/organisation.py [hunk 3]
```diff
@@ -60,3 +67,37 @@ def get_organisation_taxonomy_by_name(db: Session, org_name: str) -> TaxonomyDat
...
+    )
+    found_categories = {c[0].value: c[1] for c in counts}
+    count_by_category = {}
+
+    # Supply zeros when there aren't any
+    for category in [e.value for e in FamilyCategory]:
+        if category in found_categories.keys():
+            count_by_category[category] = found_categories[category]
+        else:
+            count_by_category[category] = 0
```
Comment (olaughter) [inline app/core/organisation.py:97]: Potential simplification here, what do you think?

```suggestion
    categories =  [e.value for e in FamilyCategory]
    count_by_category = {c[0].value: (c[1] if c[0].value in categories else 0) for c in counts}
```

**Review by olaughter — APPROVED** [review]
Nice one!

File: app/core/organisation.py (hunk 3 above)
Comment (diversemix) [reply app/core/organisation.py:97]: ok - yes its simpler ... but IMHO this is more difficult to maintain

**Pull request merged**